document/value_test.go

```diff
@@ -21,6 +21,7 @@ func TestValueString(t *testing.T) {
 		{"bool", document.NewBoolValue(true), "true"},
 		{"int", document.NewIntegerValue(10), "10"},
 		{"double", document.NewDoubleValue(10.1), "10.1"},
+		{"double with trailing .0", document.NewDoubleValue(10), "10.0"},
```

Comment:
Could you add a test with a very big float with a trailing .0?

Reply:
Done, I removed this one and replaced it with a big float that overflows an int64.

Reply:
> removed this one

I think we can keep the simple test case.